src/components/BtmNavbar.jsx

```diff
@@ -5,7 +5,7 @@ import { House,SquaresFour,Plus,User } from 'phosphor-react';
 
 const BtmNavbar = ()=>{
     return(
-        <div className='flex items-center justify-around inset-0 fixed bottom-0'>
+        <div className='flex items-center justify-around fixed bottom-0 w-full bg-white py-4 '>
             <House size={30} weight="fill" />
```

Comment:
Don't use static values. It will be difficult to maintain.

Reply:
would work on it

Reply:
Just to clarify, static value in this case is `size={30}` the 30 in this.